application/src/main/java/org/togetherjava/tjbot/commands/tags/TagsCommand.java

```diff
@@ -41,12 +45,24 @@ public TagsCommand(TagSystem tagSystem) {
 
     @Override
     public void onSlashCommand(@NotNull SlashCommandEvent event) {
-        // TODO A list might be better than comma separated, which is hard to read
+
+        ArrayList<String> list = new ArrayList<>(tagSystem.getAllIds());
+
+        if (list.size() > 200) {
+
+            Logger logger = Logger.getLogger(TagsCommand.class.getName());
+
+            logger.setLevel(Level.WARNING);
+            logger.warning("- WARNING - TAGS ARE BEYOND 200 LINES ");
```

Comment:
* dont use capslock
* dont say WARNING, it is already a warning
* the message only explains the code condition but instead it should explain what the actual problem is and why this message is here in the first place...

for example
```
The amount of tags is very high and might soon exceed the maximum character limit. The code should be adjusted to support this edge case soon.
```
together with a `// TODO Implement this edge case` next to it, so that we do not forget to track it.

Reply:
Thank you! Il update and make a PR